local-volume/provisioner/pkg/discovery/discovery_test.go

```diff
@@ -224,7 +225,7 @@ func TestDiscoverVolumes_BadVolume(t *testing.T) {
 
```

Comment:
Can you add a test case here to make sure we don't discover a volume that is being cleaned?

Reply:
This has already been implemented in TestDiscoverVolumes_CleaningInProgress